.github/workflows/run-macro-micro-dummy.yml

```diff
@@ -12,24 +12,43 @@ jobs:
     runs-on: ubuntu-latest
     container: precice/precice
     steps:
+
       - name: Checkout Repository
         uses: actions/checkout@v2      
+
       - name: Install Dependencies
         run: |
           apt-get -qq update
           apt-get -qq install python3-dev python3-pip git python-is-python3 pkg-config
           python -m pip install --upgrade pip
           pip install setuptools wheel twine
+
       - name: Install micro-manager
         run: pip3 install --user .
+
       - name: Run python macro-micro dummy
+        timeout-minutes: 3
         run: |
           cd examples/
-          python3 macro_dummy.py & python3 python-dummy/run_micro_manager.py
+          python3 python-dummy/run_micro_manager.py --config ../micro-manager-config.json & python3 macro_dummy.py
+
+      - name: Run adaptive python macro-micro dummy
+        timeout-minutes: 3
```

Comment:
This is necessary because if any of the solvers fail, the other solver is still running and thereby the action not declared a fail. See [this](https://github.com/precice/micro-manager/actions/runs/4692522581/jobs/8318340709) run. The other dummy was still waiting for the solver even though it had already failed. This is not an issue if the error happens while the solvers are running, in this case the other solver also fails.